plugins/ruby-foreman-remote-execution/debian/rules

```diff
@@ -15,3 +15,11 @@ PLUGIN_HAS_WEBPACK = true
 PLUGIN_HAS_PACKAGE_JSON = true
 
 include /usr/share/foreman/foreman-plugin.mk
+
+override_dh_auto_install:
+	cp -r ./usr/share/foreman/vendor/ruby/*/gems/foreman_remote_execution-*/extra ./
```

Comment:
I'm surprised we even ship `extra` to `/usr/share` I wouldn't expect it to because they're intended to be moved to different locations but still allow us to ship a simple gem. Not sure if you want to fix it here. For example by changing this:
```suggestion
	mv ./usr/share/foreman/vendor/ruby/*/gems/foreman_remote_execution-*/extra ./
```

Reply:
We, uh, don't.
That's `./usr`, not `/usr`.
The local copy of `/usr/share/foreman` that we use to build plugins in.
The deb ships the `.gem` and the wepack things.